app/utils/file/file_picker/index.ts

```diff
@@ -272,7 +276,7 @@ export default class FilePickerUtil {
 
         if (hasPermission) {
             try {
-                const docResponse = (await DocumentPicker.pick({allowMultiSelection, type: [fileType]}));
+                const docResponse = (await DocumentPicker.pick({allowMultiSelection, type: [fileType], copyTo: 'cachesDirectory'}));
```

Comment:
What is the below a string and not a constant like `FileSystem.CachesDirectoryPath` ?

```
 copyTo: 'cachesDirectory'}));
```

Reply:
@enahum :point_up:

Reply:
The value is a string as is required by the type definition, being a type definition you only have two valid options